### DISCUSSION TITLE
Regarding running Graylog on Unraid

### DISCUSSION THREAD

**Discussion by bluehysteric** [RobertDWhite/WhiteMatterTech#12] [General]
Regarding running Graylog on Unraid, there is currently a stable version of the software, could you update the tutorial to work with the latest version?

**Comment by RobertDWhite** [comment]
Hi @bluehysteric . For some reason, GitHub either did not notify me of this conversation or I missed it. Let me look into this and see how it goes. I will update if feasible.

Thanks!
